great-idea/index.html

```diff
@@ -15,9 +15,54 @@
 </head>
 
   <body>
+<div class="container">
+  <a href='#'>Services</a>
+  <a href='#'>Product</a>
+  <a href='#'> Vision</a>
+  <a href='#'> Features </a>
+  <a href='#'>About</a>
+  <a href='#'>Contact </a>
+</div>
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
The image here should go int the container holding the a tags if you want it to be in the same container box.